mdx-layout/src/index.tsx

```diff
@@ -13,6 +13,7 @@ export const Layout = (props) => (
         light: 'https://cdn.jsdelivr.net/npm/water.css/out/light.min.css',
         dark: 'https://cdn.jsdelivr.net/npm/water.css/out/dark.min.css',
       }}
+      onSwitch={props.onSwitch}
```

Comment:
May I ask why do you need this event ? I used this technique with StyleSheetSwitch but maybe there is a better way to do this, I'm not an expert on React but I guess with useContext() we could achieve also some kind of useful propagation of event no ? I'm looking for guidance/advice

Reply:
discussed this on discord